sebs/azure/azure.py

```
@@ -304,13 +264,112 @@ def update_function(
+                found_trigger = True
+                trigger.url = function_url
+                break
+
+        if not found_trigger:
+            trigger = HTTPTrigger(
+                function_url, self.config.resources.data_storage_account(self.cli_instance)
+            )
+            trigger.logging_handlers = self.logging_handlers
+            function.add_trigger(trigger)
+
+    def update_envs(self, function: Function, code_package: Benchmark, env_variables: dict = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)